### PR TITLE
fix(): a11y visual fixes components

### PR DESCRIPTION
**Pull request by jennmedellin**
## Description
- Increase contrast on warn color (theme.scss)
- Increase contrast on messages by removing background color 
- Increase contrast on code highlighted in markdown component
- Increase contrast on statusbar (footer text) on text editor component

#### Test Steps
- [ ] `npm run a11y`
- [ ] Expect errors (Steven is working on these code errrors)
- [ ] Verify no contrast related errors (ctrl + f > search contrast) except those already resolved in previous PRs (i.e. theme accent color contrast, top nav contrast, higlight contrast)
- [ ] `npm run serve`
- [ ] Verify design matches screenshots below

#### General Tests for Every PR

- [ ] `npm run serve:prod` still works.
- [ ] `npm run tslint` passes.
- [ ] `npm run stylelint` passes.
- [ ] `npm test` passes and code coverage is not lower.
- [ ] `npm run build:lib` still works.

##### Screenshots or link to StackBlitz/Plunker
screen shot 2019-03-05 at 3 35 16 pm
screen shot 2019-03-05 at 3 34 46 pm

### CODE REVIEW

File: src/styles.scss [hunk 1]
```diff
@@ -21,3 +21,8 @@
   border: 0;
   min-height: 700px;
 }
+
+// text editor component a11y compliance
+.editor-statusbar {
+  color: #000 !important;
```
Comment (christianmemije) [inline src/styles.scss]: This does not belong here.

File: src/app/components/components/message/message.component.html [hunk 2]
```diff
@@ -80,11 +80,11 @@
       <ng-template matTabLabel>Demo</ng-template>
       <mat-card-content>
         <p class="mat-body-1">Using <code>color</code>:</p>
-        <td-message label="Purple!" sublabel="Simply add color=blah" color="purple" icon="error">
```
Comment (christianmemije) [inline src/app/components/components/message/message.component.html]: This example should have a color prop not a class

File: src/platform/core/message/_message-theme.scss [hunk 3]
```diff
@@ -26,15 +26,12 @@
   .td-message {
     &.mat-primary {
       color: mat-color($primary);
-      background-color: mat-color($primary, 0.15);
```
Comment (christianmemije) [inline src/platform/core/message/_message-theme.scss]: I think having a background-color is useful to make them stand out. Just need to play with contrast between text-color and background-color.

**Comment by jennmedellin** [comment]
With our new design system we will be adding a high-contrast theme to address accessibility. Closing these for now.

**Pull request closed without merging**